Marlin/Configuration_adv.h

```diff
@@ -227,7 +227,6 @@
 // Define a pin to turn case light on/off
 //#define CASE_LIGHT_PIN 4
 //#define CASE_LIGHT_DEFAULT_ON   // Uncomment to set default state to on
-
```

Comment:
They're assigning a pin here rather than in a pins file.  Is that what you want?  

I can include this in the pins file changes you requested for the two spindle pins.